__tests__/database.test.js

```diff
@@ -59,13 +58,74 @@ describe("GET endpoints", () => {
     });
 });
 
+
+describe("GET articles", () => {
+    test("Request from /api/articles/1 returns an object with required values", () => {
```

Comment:
just a suggestion, but I would format the test description like "200: Request from /api/articles/1 returns an object with required values", it's easy to see at a glance what test is which